app/src/androidTest/java/ch/epfl/sdp/blindly/main_screen/my_matches/MyMatchesFragmentTest.kt

```diff
@@ -35,12 +37,12 @@ class WeatherPageTest {
     fun setup() {
         hiltRule.inject()
         runBlocking { goToProfileFragment() }
-        init()
+        Intents.init()
```

Comment:
Just for coherence but I believe we usually imported the elements and kept just init() rather than Intents.init()